src/widgets/main/company/style.module.scss

```
@@ -0,0 +1,72 @@
+@use "ds-token" as token;
+
+.company {
+	@include token.flex_between;
+	align-items: flex-start;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)